src/components/molecules/navLinks/NavLinks.tsx

```diff
@@ -3,14 +3,20 @@ import { Link } from 'react-router-dom'
 
 type NavLinksProps = {
   navLink: { route: string; label: string }
+  isSelected?: boolean
 }
 
-export function NavLinks({ navLink }: NavLinksProps) {
+export function NavLinks({ navLink, isSelected }: NavLinksProps) {
```

Comment:
🔧 If `isSelected` is optional, I'd prefer not to leave it `undefined`. In general, nothing should be `undefined` Can we set it to a value here?

```suggestion
export function NavLinks({ navLink, isSelected = false }: NavLinksProps) {
```

Reply:
I removed the option as the more I thought about it `isSelected` is needed for the styling to trigger as we desire.